.github/workflows/release.yml

```diff
@@ -31,26 +31,19 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          . .github/workflows/release_info.sh ${{ github.event.repository.name }}
+          . .github/workflows/release_info.sh ${{ github.event.repository.full_name }}
           echo ::set-output name=CHANGELOG::$(echo -e "${CHANGELOG}")
           echo ::set-output name=SKIP_RELEASE::${SKIP_RELEASE}
           echo ::set-output name=IS_CANDIDATE::${IS_CANDIDATE}
-      - name: Candidate build
```

Comment:
We aren't doing anything meaningfully different in the release between a -rc tag and a regular tag

Reply:
✂️ 🔪 💯